### PR TITLE
Implements the brokers down splash page.  

### PR DESCRIPTION
**Pull request by rfornea**
-implements the "brokers down" splash page when the user tries to click "Upload a File" from the main path choice page
-implements the "brokers down" splash page when the user tries to click "Start Upload" from the `/upload-form` page (in case they manually navigate to the `/upload-form` page in their address bar)
-does not currently implement the "brokers down" splash page when the users tries to download since downloads could still work, but leaves a TODO for us to remember to add something later to handle the case for when the tangle is down as well.  

-modifies the `execObservableIfBackendAvailable` method to call one observable if the backend is available and a different observable if it's not.  
-modifies the `goToUploadForm` method in the navigation epic to call `execObservableIfBackendAvailable` and navigate to `/upload-form` if available or `/brokers-down` if not. 

-fixes a few spelling errors

### CODE REVIEW

**Comment by rfornea** [comment]
It would appear that some files in this PR have been prettier'd that didn't get prettier'd in the last PR.  You guys want me to make a separate PR with more prettier stuff so you don't have sift through it all to see the real changes?  @AaronVasquez @EdmundMai

File: src/components/brokers-down/brokers-down-slide.tsx [hunk 1]
```diff
@@ -41,28 +41,32 @@ const BrokersDownLinkText = styled.a`
   color: #0068ea !important;
   text-decoration: none;
   ${BrokersDownLinkSpan}:hover & {
-  text-decoration: none;
-  color: #0068ea !important;
-  }`;
+    text-decoration: none;
+    color: #0068ea !important;
+  }
```
Comment (EdmundMai) [inline src/components/brokers-down/brokers-down-slide.tsx]: move this directly to the styles under `BrokersDownLinkSpan` (line 37).

it'll look like this:
```
BrokersDownLinkSpan = styled.span`
  &:hover {
    ...
  }
`
```

File: src/components/shared/slide-error.tsx [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-        <div className="container bordered indented-container">
+  return (
+    <section className="slide">
+      <div className="container bordered indented-container">
         <div className="slide-body slide-error-body">
-        <h1 className="slide-title">{title}</h1>
-        <hr className="underline" />
-        {children}
+          <h1 className="slide-title">{title}</h1>
+          <hr className="underline" />
```
Comment (EdmundMai) [inline src/components/shared/slide-error.tsx]: can we styled-componentize this stuff too?

File: src/components/brokers-down/brokers-down-slide.tsx (hunk 1 above)
Comment (rfornea) [reply src/components/brokers-down/brokers-down-slide.tsx]: Strangely, this did not work.

File: src/components/shared/slide-error.tsx (hunk 2 above)
Comment (rfornea) [reply src/components/shared/slide-error.tsx]: Will do.

**Pull request merged**